RevenueCat/Scripts/Purchases.cs

```diff
@@ -134,10 +154,17 @@ private void Setup(string newUserId)
             .SetUseAmazon(useAmazon)
             .SetDangerousSettings(dangerousSettings);
 
-        Setup(builder.Build());
+        Configure(builder.Build());
     }
 
+    [Obsolete("Deprecated, use Configure instead.", false)]
```

Comment:
this method was never public in a final release, but it doesn't hurt to have the deprecation for anyone using a release candidate

Reply:
Well... I guess the only downside is that we will have an extra deprecated function to test/remove at some point... But then again, I think what you said makes sense so I'm ok with this 👍